src/functions/durableOrchestration.ts

```diff
@@ -0,0 +1,25 @@
+import { HttpRequest, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { DurableOrchestrationClient, IOrchestrationFunctionContext } from 'durable-functions/lib/src/classes';
+
+df.orchestration('DurableFunctionsOrchestratorJS', function* (context: IOrchestrationFunctionContext) {
```

Comment:
We shouldn't be using interfaces starting with `I` in our templates

Reply:
Yeah I agree. Again, this is the type that's available in the Durable SDK and there isn't another type/interface that doesn't have the `I` to describe an orchestration context